modules/microshift-greenboot-how-workload-health-check-scripts-work.adoc

```diff
@@ -0,0 +1,21 @@
+//Module included in the following assemblies:
+//
+//* microshift_running-applications/microshift-greenboot-workload-scripts.adoc
+
+:_content-type: CONCEPT
+[id="microshift-greenboot-how-workload-health-check-scripts-work_{context}"]
+= How workload health check scripts work
+
+The workload or application health check script described in this tutorial uses the {product-title} health check functions that are available in the `/usr/share/microshift/functions/greenboot.sh` file. This enables you to reuse procedures already implemented for the {product-title} core services.
+
+The script starts by running checks that the basic functions of the workload are operating as expected. To run the script successfully:
+
+* Execute the script from a root user account.
+* Enable the {product-title} service.
+
+The health check performs the following actions:
+
+* Gets a wait timeout of the current boot cycle for the `wait_for` function.
+* Calls the `namespace_images_downloaded` function to wait until pod images are available.
+* Calls the `namespace_pods_ready` function to wait until pods are ready.
+* Calls the `namespace_pods_not_restarting` function to verify pods are not restarting. Restarting pods can indicate a crash loop.
```

Comment:
The second sentence would maybe be better suited in a [NOTE]. `Restarting pods can indicate a crash loop`.